addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -402,7 +402,7 @@ def _process_step_forward_operator(self, discuss_channel, users=None):
                         [
                             self.env.user.display_name
```

Comment:
Not related to this PR, but this condition makes little sense, it assumes the forward is always triggered by the current user, but there is no reason to be so strict.

I think it would be simpler to just compute the name based on all the current members of the channel.